command/sub8_launch/launch/subsystems/cameras/calibration/00b09d0100ea51be.yaml

```diff
@@ -1,6 +1,6 @@
 image_width: 644
 image_height: 482
-camera_name: 00b09d0100ea51be
+camera_name: 15356350
```

Comment:
I've rebooted several times w/o any changes. If they did, you could get the updated serial number by parsing the output of `rosrun pointgrey_camera_driver list_cameras`.